File: 0058-length-of-last-word/0058-length-of-last-word.ex
defmodule Solution do
  @spec length_of_last_word(s :: String.t) :: integer
  def length_of_last_word(s) do
    String.trim_trailing(s) 
    |> String.split(" ") 
    |> List.last() 
    |> String.length()
  end
end

File: 0058-length-of-last-word/0058-length-of-last-word.kt
class Solution {
    fun lengthOfLastWord(s: String): Int {
        var len = 0
        for (i in s.indices) {
            if (len != 0 && s[s.length-1-i] == ' ') break
            if (s[s.length-1-i] == ' ') continue
            ++len
        }
        return len
    }
}